packages/receipts/package.json

```
@@ -1,11 +1,11 @@
 {
   "name": "@graphprotocol/receipts",
   "version": "0.0.1",
-  "description": "Graph Network State Channel Receipt Manager",
+  "description": "State Channel Receipt Manager",
```

Unintentional change
```suggestion
  "description": "Graph Network State Channel Receipt Manager",
```